src/views/Privacy.vue

```diff
@@ -22,7 +22,7 @@
 
           <p>
             This Privacy Policy (“Policy”) is applicable for the Superhero.com created
-            and operated Website http://youdonotneedacapetobeahero.com/ (the “Website”),
+            and operated Website http://superhero.com/ (the “Website”),
```

Comment:
```suggestion
            and operated Website https://beta.superhero.com/ (the “Website”),
```

Should be **beta** for now.